validator/client/propose.go

```diff
@@ -177,11 +177,21 @@ func (v *validator) ProposeBlock(ctx context.Context, slot primitives.Slot, pubK
 		trace.Int64Attribute("numAttestations", int64(len(blk.Block().Body().Attestations()))),
 	)
 
+	if err := logProposedBlock(blk, blkResp.BlockRoot); err != nil {
+		log.WithError(err).Error("Failed to log proposed block")
+	}
+
+	if v.emitAccountMetrics {
+		ValidatorProposeSuccessVec.WithLabelValues(fmtKey).Inc()
```

Comment:
can this trigger if the logProposerBlock produced an error above?

Reply:
Not before, but it should trigger. `ValidatorProposeSuccessVec` should increment even if somewhere we fail to log